tutorials/plot_background_statistics.py

```diff
@@ -383,8 +392,8 @@ def plot_t_p(t, p, title, mcc, axes=None):
 # made, as the entire cluster as a whole is used to reject the null.
 # Moreover, because the test statistic concerns the full data, the null
 # hypothesis (and our rejection of it) refers to the structure of the full
-# data. For more information, see also the
-# `excellent FieldTrip cluster interpretation tutorial <ft_cluster>`_.
+# data. For more information, see also the comprehensive
+# `FieldTrip tutorial <http://www.fieldtriptoolbox.org/faq/how_not_to_interpret_results_from_a_cluster-based_permutation_test>`_.
```

Comment:
revert linking method change

Reply:
I changed the links because they didn't work in master: https://martinos.org/mne/stable/auto_tutorials/plot_background_statistics.html

I can remove the `# noqa`, but then PEP8 complains.